scripts/actions/spells/white/raise.lua

```diff
@@ -3,33 +3,41 @@
 -----------------------------------
 ---@type TSpell
 local spellObject = {}
+local BEHAVIOR_RAISABLE = 0x004
```

Comment:
This is unneeded. We already have a global, predefined enum for this
`xi.behavior.RAISABLE`